src/index.js

```diff
@@ -107,13 +106,17 @@ function start() {
   player = new BrowserWindow({
     width: 427,
     height: 300,
+    minWidth: 427,
+    minHeight: 300,
     closable: false,
     useContentSize: true,
     title: 'Headset - Player',
     icon: windowIcon,
     webPreferences: { nodeIntegration: true },
   });
 
+  player.minimize();
```

Comment:
I like this idea, but this is not working on Linux. It seems that this needs to be run after `player.loadURL` is called, otherwise, it doesn't work, or inside the `did-finish-load` event for the player.